packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -431,4 +427,32 @@ class DefaultTextEditingShortcuts extends Shortcuts {
         return _windowsShortcuts;
     }
   }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget result = child;
+    if (kIsWeb) {
+      // On web platform, these shortcuts makes sure the following:
+      //
+      // 1. Shortcuts fired when a TextField is focused are ignored and
+      //    forwarded to the browser.
+      // 2. Shortcuts fired when no TextField is focused will still trigger
+      //    default text editing Shortcuts
+      //
+      // In (2), the disabling shortcuts will still fire; however, the
+      // [DoNothingAndStopPropagationTextIntent] will not be handled because the
+      // context is above the TextField. The key event will then trigger the
+      // default Text Editing Shortcuts.
+      result = Shortcuts(
+        debugLabel: '<Web disabling text shortcuts>',
```

Comment:
Nit: "Web Disabling Text Editing Shortcuts"?

Looking at other debugLabels in the codebase, most capitalize all of the first letters like this, but not all...